packages/common/client-utils/src/layerGenerationState.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Policy check is failing because this line ends with a full stop.
@scottn12 @tylerbutler Not sure if this is intentional but either way, this will need a build-tools change.

Reply:
Adding `layerGenerationState.ts` to `policy:handlerExclusions` seems to fix the problem. However, I think it might be better to just update the code that generates the file to not have a full stop. What do you think?